Src/BootCamp.Chapter/BalanceStats.cs

```diff
@@ -7,31 +7,217 @@ public static class BalanceStats
         /// </summary>
         public static string FindHighestBalanceEver(string[] peopleAndBalances)
         {
-            return "";
+            if (peopleAndBalances == null || peopleAndBalances.Length == 0)
+            {
+                return "N/A.";
+            }
+
+            
```

Comment:
3 blank lines for now reason. Please remove.

Reply:
Removed.